contrib/packs/actions/restart_component.yaml

```diff
@@ -10,7 +10,7 @@
       immutable: true
     cmd:
       immutable: true
-      default: "st2ctl restart-component {{servicename}}"
+      default: "service {{servicename}} restart"
```

Comment:
Probably description:
```yaml
description: "Action that restarts st2 service."
```
needs updating too, since something like this is legit now:
```sh
st2 run restart_component servicename=cron
```